spec/features/admin/user_spec.rb

```diff
@@ -6,7 +6,7 @@
 
     before(:each) do
       visit '/users/sign_in'
-      within('form') do
+      within('.container-fluid:nth-child(2) form') do
```

Comment:
This looks ugly, let's search for form by id or class, smth like
`within 'form#new_user' do`